lib/lookup.json

```
@@ -604,6 +604,13 @@
     "maintainers": ["bcoe", "addaleax"],
     "head": true
   },
+  "yarn": {
+    "maintainers": ["arcanis", "merceyz", "paul-soporan", "larixer"],
+    "yarn": true,
```

```suggestion
    "yarn": true,
    "install": ["install", "--mode", "skip-build"],
```
We don't need any of the native dependencies in order to run the tests so we can skip all `postinstall` build scripts.